arvo/clay/data-types.md

```diff
@@ -186,28 +217,62 @@ if `q:ankh` is null), then this produces null.
 
 `%z` requests the `++ankh` of the specified commit at the specfied path.
 
-### `++arch`, shallow filesystem node
+#### `$ankh`, filesystem node
 
 ```hoon
-    ++  arch  ,[p=@uvI q=(unit ,@uvI) r=(map ,@ta ,~)]      ::  fundamental node
++$  ankh                                                ::  expanded node
+  $~  [~ ~]
+  $:  fil=(unit [p=lobe q=cage])                        ::  file
+      dir=(map @ta ankh)                                ::  folders
+  ==                                                    ::
 ```
 
-This is analogous to `++ankh:clay` except that the we have neither our
-contents nor the ankhs of our children. The other fields are exactly the
-same, so `p` is a hash of the associated ankh, `u.q`, if it exists, is a
-hash of the contents of this node, and the keys of `r` are the names of
-our children. `r` is a map to null rather than a set so that the
-ordering of the map will be equivalent to that of `r:ankh`, allowing
-efficient conversion.
+This is a single node in the filesystem. This may be file or a directory
```

Comment:
I wouldn't describe this as a single node.  It includes all its subdirectories recursively -- see the `dir=(map @ta ankh)` type recursion.  `$arch` is indeed a single node, but `$ankh` is not.

Reply:
I'd say an `$ankh` is a recursive type that can describe a whole tree of folders and files.